modules/gateway/versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.5.0"
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">=3.71.0"
```

Comment:
we have mismatch in the version in different `versions.tf` files, should be the same everywhere IMHO

Reply:
Updated